insights/config/specs.py

```diff
@@ -239,9 +241,9 @@
     "modinfo"                   : CommandSpec("/usr/sbin/modinfo {module}", module=r"\S+"),
     "modprobe.conf"             : PatternSpec(r"etc/modprobe\.conf"),
     "modprobe.d"                : PatternSpec(r"etc/modprobe.d/.*\.conf"),
-    "mongod_conf"               : First([PatternSpec("etc/mongod.conf"),
-                                    PatternSpec("etc/mongodb.conf")]),
-    "rh_mongodb26_conf"         : PatternSpec("etc/opt/rh/rh-mongodb26/mongod.conf"),
+    "mongod_conf"               : All([SimpleFileSpec("etc/mongod.conf"),
+                                    SimpleFileSpec("etc/mongodb.conf"),
+                                    SimpleFileSpec("etc/opt/rh/rh-mongodb26/mongod.conf")]),
```

Comment:
These modifications shouldn't be here in your PR, you might need to rebase your branch to the 1.x.